### PR TITLE
Onboarding-oriented landing page, product-oriented headers, topical gentle intros

### PR DESCRIPTION
**Pull request by symbolpunk**
new:
 - onboarding-optimized landing page
 - header section ordering tweaks: `build dapps` -> `bridge` -> `node` -> `L3 chains` -> `Learn more`
 - removed "get started" section to simplify header
 - merged "non-bridge user content" into "Learn more"
 - "Arbitrum Dev Center" -> "Arbitrum Docs"
 - Landing page CTAs / icons updated to include new quickstart + troubleshooting page
 
 preview: [link](https://nitro-docs-git-onboarding-flows-lp-clarity-offchain-labs.vercel.app/)

### CODE REVIEW

File: arbitrum-docs/for-devs/gentle-introduction-dapps.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+title: "A gentle introduction: Arbitrum DApps"
+description: "Arbitrum is a suite of Ethereum scaling solutions that includes Arbitrum One, Arbitrum Nova, Stylus, and the Arbitrum Bridge. This guide introduces you to decentralized apps on Arbitrum: what they are, how they work, and the value that they provide."
+---
+
+import PublicPreviewBannerPartial from '../partials/_public-preview-banner-partial.md'; 
+
+<PublicPreviewBannerPartial />
+
+import GentleIntroPartial from '../partials/_gentle-intro-partial.md';
```
Comment (symbolpunk) [inline arbitrum-docs/for-devs/gentle-introduction-dapps.md:10]: Intent:

image

File: website/docusaurus.config.js [hunk 2]
```diff
@@ -198,9 +207,7 @@ const config = {
       },
       announcementBar: {
         id: "support_us",
-        content: `✨ Arbitrum One and Arbitrum Nova are now under decentralized governance. Learn more about Arbitrum DAO <a rel="noopener noreferrer" href="https://docs.arbitrum.foundation/" target="_blank">here</a>. ✨`,
-        backgroundColor: "rgb(8 53 117)",
-        textColor: "white",
+        content: `Arbitrum One and Arbitrum Nova are now under decentralized governance. Learn more about Arbitrum DAO <a href='https://docs.arbitrum.foundation/gentle-intro-dao-governance'>here</a>.`,
```
Comment (symbolpunk) [inline website/docusaurus.config.js:210]: Aligning with Portal copy

File: website/sidebars.js [hunk 3]
```diff
@@ -184,259 +262,215 @@ const sidebars = {
...
+          label: "FAQ index"
         },
         {
           type: "doc",
-          id: "proving/wasm-to-wavm",
-          label: "Wasm To WAVM"
+          id: "faqs/gas-faqs",
+          label: "Gas"
         },
         {
```
Comment (symbolpunk) [inline website/sidebars.js:425]: Intent is to move (/partialize) this as part of a separate FAQ organization PR

File: website/src/components/HomepageFeatures/index.tsx [hunk 4]
```diff
@@ -15,45 +15,42 @@ type FeatureItem = {
...
+    href: "/intro",
+    description: <> Learn about Arbitrum's core technologies and product portfolio.</>,
   },
   {
-    title: "For Developers",
-    Svg: require("@site/static/img/builder.svg").default,
-    href: "/getting-started-devs",
-    description: <>Guides for Dapp Devs and Node Runners</>,
+    title: "Quickstart: Build a dApp",
+    Svg: require("@site/static/img/cupcake.svg").default,
```
Comment (symbolpunk) [inline website/src/components/HomepageFeatures/index.tsx:25]: cc0 (creative commons) cupcake

File: arbitrum-docs/for-devs/gentle-introduction-dapps.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+
+<GentleIntroPartial />
```
Comment (symbolpunk) [inline arbitrum-docs/for-devs/gentle-introduction-dapps.md:12]: The `GentleIntroPartial`, as it stands, isn't _specifically about_ building dapps, so its content feels a little out of place here, but this is temporary. We can hide it with an "under construction" banner like the other gentle intro stubs, if we prefer to wait until the dev-specific gentle intro is ready.

**Review by DZGoldman — APPROVED** [review]
looks slick, lgtm. (my only _very_ minor comment is I kinda want "quickstart node" and "troubleshoot your node" icons to look different, but tbh what do i know)

**Pull request merged**